modules/vaos/app/controllers/vaos/v2/referrals_controller.rb

```diff
@@ -64,11 +67,32 @@ def referral_mode_param
         params.fetch(:mode, 'C')
       end
 
+      # CCRA Referral Status Codes:
+      # S  - Suspend: Referral temporarily paused/on hold
+      # BP - EOC Complete: Episode of Care is completed
+      # AP - Approved: Referral approved/authorized for care
+      # A  - First Appointment Made: Initial appointment scheduled
+      #
+      # TODO:
+      # I  - Unknown - Possibly means Initial: Referral initiated/in progress
+      # AC - Unknown - Possibly means Appointment Canceled
+      #
       # The referral status parameter for filtering referrals
       # @return [String] the referral status
       def referral_status_param
-        # TODO: Need to verify what statuses we can allow. API spec is not clear.
-        params.fetch(:status, "'S','BP','AP','AC','A','I'")
+        # Default to only show referrals that a veteran can make appointments for.
+        params.fetch(:status, "'AP','AC','I'")
+      end
+
+      # Filters out referrals that have expired (expiration date before today)
+      #
+      # @param referrals [Array<Ccra::ReferralListEntry>] The collection of referrals
+      # @return [Array<Ccra::ReferralListEntry>] Filtered collection without expired referrals
+      def filter_expired_referrals(referrals)
+        return [] unless referrals.respond_to?(:reject)
```

Comment:
having a guard here is a good idea, but I think defaulting to an array is in violation of the RDOC above and can mask a silent failure, might be better to raise an argument error to fail loudly:

```suggestion
         raise ArgumentError, 'referrals must be an enumerable collection' unless referrals.respond_to?(:each)
```